<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNodeWindow</class>
 <widget class="QDialog" name="AddNodeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabNodeNumber">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabNodeSet">
      <attribute name="title">
       <string>Set of Nodes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="9" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Production Period MAX</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Production Period STEP</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QTabWidget" name="tabNodeSetArrange">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabNodeSetArrangeSquare">
          <attribute name="title">
           <string>Square</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Nodes on Side MIN</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QDoubleSpinBox" name="setNodeSquareDistance">
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QSpinBox" name="setNodeSquareSideMin">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Node Distance</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Nodes on Side MAX</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QSpinBox" name="setNodeSquareSideMax">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>6</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabNodeSetArrangeLine">
          <attribute name="title">
           <string>Line</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="3" column="0">
            <widget class="QDoubleSpinBox" name="setNodeLineLength"/>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Line Length</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QSpinBox" name="setNodeLineNumber"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Node Number MIN</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QSpinBox" name="setNodeLineNumber_2"/>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Node Number MAX</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabNodeSetArrangeEllipse">
          <attribute name="title">
           <string>Ellipse</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QSpinBox" name="setNodeEllipseNumber"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Node Number MIN</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QDoubleSpinBox" name="setNodeEllipseWidth"/>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Ellipse Width</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QDoubleSpinBox" name="setNodeEllipseHeight"/>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Ellipse Height</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Node Number MAX</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QSpinBox" name="setNodeEllipseNumber_2"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QSpinBox" name="setNodePeriodMin">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>70000</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QSpinBox" name="setNodePeriodStep">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>70000</number>
         </property>
         <property name="value">
          <number>250</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Production Period MIN</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Node Radius</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QDoubleSpinBox" name="setNodeRadius">
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="setNodeTraces">
         <property name="text">
          <string>netInterfaces traces activated</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QSpinBox" name="setNodePeriodMax">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>70000</number>
         </property>
         <property name="value">
          <number>25000</number>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QSpinBox" name="setNodeRuns">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Runs</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNodeSingle">
      <attribute name="title">
       <string>Single Node</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QDoubleSpinBox" name="singleNodeRadius"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Node Radius</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="singleNodePosX"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Position X</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QDoubleSpinBox" name="singleNodePosY"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Position Y</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddNodeWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddNodeWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
